backend/include/Helpers.hpp

```diff
@@ -41,6 +42,30 @@ class Helpers {
         std::string temp = std::to_string(frequencyHz / 1000);
         return temp.substr(0, 3) + "." + temp.substr(3, 7);
     }
+
+    /**
+     * @brief Converts a JSON property that is either true, false, or "toggle" to a boolean value.
+     *
+     * @param incomingValue The JSON property to convert.
+     * @param currentValue The current value of the property.
+     * @return The converted boolean value, either the value specified in the JSON property, or the
+     *        opposite of the current value if the property is "toggle".
+     */
+    inline static bool ConvertBoolOrToggleToBool(
```

Comment:
This is used to process all of the properties sent in the websocket request to update rx, tx, xc, xca, headset.